pkgs/top-level/linux-kernels.nix

```diff
@@ -488,7 +488,7 @@ in {
 
     rust-out-of-tree-module = if lib.versionAtLeast kernel.version "6.7" then callPackage ../os-specific/linux/rust-out-of-tree-module { } else null;
 
-    tuxedo-keyboard = if lib.versionAtLeast kernel.version "4.14" then callPackage ../os-specific/linux/tuxedo-keyboard { } else null;
+    tuxedo-driver = if lib.versionAtLeast kernel.version "4.14" then callPackage ../os-specific/linux/tuxedo-driver { } else null;
```

Comment:
As you renamed the package here, could you add an alias to block at line 608, in-case someone didn't use the module.
Something like this:
```
tuxedo-keyboard = self.tuxedo-drivers;
```